DataSourceKitTests/Example/CTableViewCell.swift

```diff
@@ -12,7 +12,7 @@ import DataSourceKit
 class CTableViewCell: UITableViewCell {
     var idLabel = UILabel(frame: .zero)
     
-    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
```

Comment:
This change is applied in #13, so please update this branch. Sorry for my late response 🙇